fuzzing/nxt_http_controller_fuzz.c

```diff
@@ -84,6 +80,20 @@ LLVMFuzzerTestOneInput(const uint8_t *data, size_t size)
     nxt_main_log.level = NXT_LOG_ALERT;
     r_controller->conn->log = nxt_main_log;
 
+    /*
+     * Process request and fields.
+     */
+    nxt_memzero(&rp, sizeof(nxt_http_request_parse_t));
+
+    if (nxt_http_parse_request_init(&rp, mp) != NXT_OK) {
+        goto failed;
+    }
```

Comment:
Can you do this same way as the one below? I.e.

```c
rc = nxt_http_parse_request_init(&rp, mp);
...
```

Rather than having two different styles...